UNITTESTS/empty_baseline/unittest.cmake

```diff
@@ -1,57 +1,55 @@
-
 ####################
 # UNIT TESTS
 ####################
 
-set(unittest-includes ${unittest-includes}
-  .
-  ..
-  ../connectivity/mbedtls/include/mbedtls/
-  ../connectivity/mbedtls/platform/inc/
-  ../platform/mbed-trace/include/mbed-trace/
+set(unittest-includes
```

Comment:
@rajkan01 how does this affect your upcoming unittests PR ? Both PRs should be part of the next release.